addons/l10n_de/__manifest__.py

```diff
@@ -5,7 +5,6 @@
     'name': 'Germany - Accounting',
     "version": "2.0",
     'author': 'openbig.org',
-    'version': '1.1',
```

Comment:
🤦🏼‍♂️

Reply:
@odoo/rd-runbot can't a check be added (with pylint?) to avoid this kind of issue to happen again?

Reply:
> @odoo/rd-runbot can't a check be added (with pylint?) to avoid this kind of issue to happen again?

Yes, it's doable with pylint (rule `duplicate-key`): https://runbot.odoo.com/runbot/build/49230948